.github/workflows/deploy-to-production-stage-2.yml

```diff
@@ -132,18 +132,6 @@ jobs:
           id=$(echo '${{steps.files.outputs.details}}' | sed -e 's/+00:00/Z/g' | jq ' . | map( select( .name | startswith( "${{ env.TARGET }}-system-" ) ) ) | sort_by( .createdAt | fromdate ) | reverse | .[0].id')
           echo "id=${id}" >> $GITHUB_OUTPUT
 
-      - name: Get Lastest Platform Properties for Envrinment
```

Comment:
Platform properties is a generic file that is needed for other things too. If your platform.properties file is empty you can remove this, but I suspect you might need to re-add this line in the future